test/conformance/testdata/conformance.yaml

```diff
@@ -2090,6 +2090,39 @@
     support running as UID / GID, or privilege escalation.'
   release: v1.15
   file: test/e2e/common/node/security_context.go
+- testname: Sysctls, prevent launching greylisted sysctl
+  codename: '[sig-node] Sysctls [LinuxOnly] [NodeFeature:Sysctls] should not launch
```

Comment:
just recording notes as part of review.

this test checks for the following sysctl: "kernel.msgmax"

if we add this test to conformance, and this is later supported as a safe systctl by being namespaced, what is the policy for changing the enumerated sysctl in future?

runc see:
https://github.com/opencontainers/runc/blob/09523b79d01d032d2faeeb7eef4ad5a31eda528f/libcontainer/configs/validate/validator.go#L138

My memory is not as strong in this area as it should be but i am biased to not promote tests related to greylisting sysctls at this time, unless there is a particular policy on above question we accounted.